test/cases/html-web-components/expected.html

```diff
@@ -1,7 +1,7 @@
 <wcc-picture-frame title="Greenwood">
   <div class="picture-frame">
-    <img src="https://www.greenwoodjs.io/assets/greenwood-logo-og.png" alt="Greenwood logo">
-    <br>
+    <img src="https://www.greenwoodjs.io/assets/greenwood-logo-og.png" alt="Greenwood logo" />
+    <br />
```

Comment:
hmm, I thought the issue was that the **expected** would be _without_ the closing `/` for these void elements?

I can understand if we start with `<img/>` but end up with `<img>` but shouldn't parse5 be stripping the closing `/` and thus the expected should be `<img>`?